src/main/scala/mesosphere/marathon/core/heartbeat/HeartbeatActor.scala

```diff
@@ -0,0 +1,97 @@
+package mesosphere.marathon.core.heartbeat
+
+import akka.actor._
+import scala.concurrent.duration._
+
+// HeartbeatActor monitors the heartbeat of some process and executes handlers for various conditions.
```

Comment:
because I haven't memorized scaladoc conventions yet. I can take a stab at
this. Last time I tried to do this sbt complained about something I didn't
understand, and .. time...

On Fri, Jul 15, 2016 at 2:23 PM, Matthias Eichstedt <
notifications@github.com> wrote:

> In src/main/scala/mesosphere/marathon/core/heartbeat/HeartbeatActor.scala
> <https://github.com/mesosphere/marathon/pull/4069#discussion_r71018734>:
>
> > @@ -0,0 +1,97 @@
> > +package mesosphere.marathon.core.heartbeat
> > +
> > +import akka.actor._
> > +import scala.concurrent.duration._
> > +
> > +// HeartbeatActor monitors the heartbeat of some process and executes handlers for various conditions.
>
> Is there a reason why you didn't use scaladoc syntax? not that we're
> consistent with that ..
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4069/files/718c11edd708bc282deea7cca641f663168b5c7c#r71018734>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLBQlG6BCUOLxhCwpurqPgjhWk40lks5qV9AJgaJpZM4JGoyy>
> .
>